src/BuildIt.Synchronization/BuildIt.Synchronization/SynchronizationContext.cs

```diff
@@ -22,6 +22,7 @@ public class SynchronizationContext<TSynchronizationStages> :
         ISynchronizationContext<TSynchronizationStages>
         where TSynchronizationStages : struct
     {
+        private const double OneHundred = 100;
```

Comment:
Hmmm, why? is there a reason we need 0-100 instead of just 0-1? 0-1 is much better for use in calculations around scale /progress etc

Reply:
if you look at the string with the progress it actually has x% so if we're displaying in that format, it should really be 0-100

Reply:
yep ok, leave as is for the moment - more keen to get the build out